Views/InputsView.swift

```diff
@@ -591,7 +590,7 @@ class InputsView: UIView, UITableViewDataSource, UITableViewDelegate, UISearchBa
     switch segmentedControl.selectedSegmentIndex {
     case 0:
       createSeed(variety: seedView.varietyTextField.text!, specie: seedView.specieButton.titleLabel!.text!)
-      seedView.specieButton.setTitle("Abricotier", for: .normal)
+      seedView.specieButton.setTitle("PRUNUS_ARMENIACA".localized, for: .normal)
```

Comment:
Why specifically "PRUNUS_ARMENIACA"? Is it just because it's the first of the list?